honeybee_schema/radiance/properties.py

```diff
@@ -0,0 +1,173 @@
+"""Properties Schema"""
+from pydantic import Field, constr, validator, root_validator
+from typing import List, Union
+
+from .._base import IDdBaseModel
+from .modifier import Void, ModifierBase
+from .modifierset import ModifierSet
+
+
+class _PropertiesBaseAbridged(IDdBaseModel):
+    """Base class of Abridged Radiance Properties."""
+
+    modifier: str = Field(
+        default=None,
+        min_length=1,
+        max_length=100,
```

Comment:
The identifier here isn't subject to the same restrictions as honeybee-energy identifiers. It only has to abide by Radiance rules for identifiers. So it can be longer than 100 characters but it can't have any characters other than letters, numbers, `-` or `_`. You should make a base class like what you see here in the energy folder:
https://github.com/ladybug-tools/honeybee-schema/blob/master/honeybee_schema/energy/_base.py#L8-L32

I should have mentioned this in the last PR since you should have all of the Radiance modifiers and the properties classes inherit from this class rather than `IDdBaseModel`.
Then, you have all of the Modifie

Reply:
@chriswmackey 

I've added an `IDdRadianceBaseModel` (which inherits from `NoExtraBaseModel` to the modifiers, but I can't find an identifier or display_name property for the radiance properties: 
https://github.com/ladybug-tools/honeybee-radiance/blob/6e73b338c4b43135caaa7826d9688d9f7526fddc/honeybee_radiance/properties/_base.py#L9-L29

Additionally, it looks like properties are not extensible w/ extra keys. So shouldn't the Properties just inherit from BaseModel?

Reply:
Thank you for adding the `IDdRadianceBaseModel` and having the modifiers inherit from it. 

And you are right that I made a mistake. None of the properties classes have an identifier since they always rely on the host object (Aperture, Face, Shade, etc) to identify them. However, the RadianceProperties classes should inherit from the `NoExtraBaseModel` (NOT `BaseModel`). That docstring there only refers to the "core honeybee" properties, under which the EnergyProperties and RadianceProperties sit. Like what you see here:
https://github.com/ladybug-tools/honeybee-schema/blob/master/honeybee_schema/model.py#L68-L74
Maybe we should revise that docstring to be more specific.